src/Resources/views/Shop/Product/Show/_addToCartGiftCardInformation.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if form.giftCardInformation is defined %}
+    {% set giftCardInformationForm = form.giftCardInformation %}
+    {% if giftCardInformationForm.amount is defined %}
+        {{ form_row(giftCardInformationForm.amount) }}
+    {% endif %}
+    {{ form_row(giftCardInformationForm.customMessage) }}
+{% endif %}
```

Comment:
This needs to be setup in the end application I suppose? If yes, then it should be in the readme

Reply:
We are telling user to copy all templates. Not sure that it is needed to clearly indicate that change.

In some changelog or upgrade guide perhaps ?

Reply:
Yeah, I think we should have an UPGRADE.md file really with all the changes we introduce